pints/_mcmc/_differential_evolution.py

```diff
@@ -308,14 +305,15 @@ def tell(self, proposed_log_pdfs):
             self._proposed = None
 
             # Return first samples for chains
-            return self._current
+            accepted = np.array([True] * self._n_chains)
+            return self._current, self._current_log_pdfs, accepted
 
         # Perform iteration
         next = np.array(self._current, copy=True)
         next_log_pdfs = np.array(self._current_log_pdfs, copy=True)
 
         # Sample uniform numbers
-        u = np.log(np.random.uniform(size=self._chains))
+        u = np.log(np.random.uniform(size=self._n_chains))
 
         # Get chains to be updated
         i = u < (proposed_log_pdfs - self._current_log_pdfs)
```

Comment:
Rename `u` and `i` to something else...?

Reply:
Separate ticket :D